cdk/bin/main.ts

```diff
@@ -27,7 +27,7 @@ const auth = new AuthStack(app, "Auth", {
 
 const backend = new BackendStack(app, "Backend", {
   stackName: stackPrefix.concat("-Backend"),
-  userPoolArn: auth.userPoolArn,
+  auth,
```

Comment:
I'm not sure if it's a good idea to pass the entire stack. It creates tighter coupling than there should be. Why not passing just the UserPool object?

Reply:
We dont have an userPool object, the alternative is to pass:

`userPool: {
    id: auth.userPoolId,
    arn: auth.userPoolArn,
}`

or the two separate strings